lib/cib/cib_ops.c

```diff
@@ -443,6 +443,11 @@ update_cib_object(xmlNode * parent, xmlNode * update)
 
     copy_in_properties(target, update);
 
+    if (xml_acl_denied(target)) {
+        crm_notice("Cannot update <%s id=%s>", crm_str(object_name), crm_str(object_id));
```

Comment:
Let's improve the user-friendliness of this message, something like:

     Cannot update <%s%s%s>: Access denied by ACLs
     object_name, (object_id? " id=" : ""), (object_id? object_id : "")

(object_name is guaranteed non-NULL at this point)